cmd2/utils.py

```diff
@@ -317,6 +344,11 @@ def clear(self) -> None:
         """Clear the internal contents"""
         self.buffer.byte_buf = b''
 
+    @staticmethod
+    def isatty() -> bool:
+        """StdSim will never be considered an interactive stream"""
+        return False
```

Comment:
I should probably change this to:
```Python
@staticmethod
def isatty() -> bool:
    if self.echo:
        return self.inner_stream.isatty()
    else:
        return False
```
@tleonhardt Thoughts?

Reply:
Yeah that probably makes sense, but if you do that make sure to update the comment 😀